Signal/src/view controllers/MessagesViewController.m

```diff
@@ -102,6 +102,13 @@ @interface UINavigationItem () {
 
 @implementation MessagesViewController
 
+- (void)dealloc {
+    // Last ditch effort to remove any sloppy lingering observers.
```

Comment:
I would remove this comment block.  This is an obvious and common call in `dealloc`.